src/plugin/ogc/wfs/wfslayerconfig.js

```diff
@@ -347,10 +351,22 @@ class WFSLayerConfig extends AbstractDataSourceLayerConfig {
     }
 
     if (options['mappings']) {
-      importer.setMappings(/** @type {Array<IMapping>} */ (options['mappings']));
+      importer.setUserMappings(/** @type {Array<IMapping>} */ (options['mappings']));
     }
   }
 
+
+  /**
+   * Returns if a mapping is elliptical
+   * @param {IMapping} mapping
+   * @return {boolean}
+   */
+  isEllipticalMapping(mapping) {
+    const field = mapping.toField;
+    return RadiusMapping.REGEX.test(field) ||
```

Comment:
Can you compare `mapping.getId()` to the `ID` for each of these mappings? Running a regex on each seems more expensive than necessary. Similar to the `updateMappings` function in the ellipse columns controller.